catroid/src/main/res/values/strings.xml

```diff
@@ -587,6 +587,11 @@
     <string name="brick_when">When tapped</string>
     <string name="brick_wait">Wait</string>
     <string name="brick_when_nfc">When NFC</string>
+    <string name="brick_stop_script">Stop script/s</string>
```

Comment:
Yes, your're right, i had this spelling the first time, but the brackets in the string didn't work in the testcases